### ISSUE TITLE
Glitchy display of desert inn sign

### ISSUE THREAD

**Issue by Allofich** [afritz1/OpenTESArena#109]
I found a desert inn sign that has a glitchy-looking display for its lower half in OpenTESArena.

It looks a little strange in the original game, too, like the lower half is a floating stone wall.

OpenTESArena
![opentesarena](https://user-images.githubusercontent.com/19624336/36028874-15cab61a-0de4-11e8-8ad6-4f0baf7f08f9.jpg)

Original Arena
![arena](https://user-images.githubusercontent.com/19624336/36028876-17e31fd2-0de4-11e8-9fdd-311631c0202f.jpg)

The two pictures are looking at the sign from opposite sides so the direction is reversed, but they both look as they do (colored dots in OpenTESArena, a stone wall in original Arena) whichever direction you look at them from.

**Comment by Carmina16** [comment]
That sprite has an unusual size, not 64 x 64, which the game expects.

**Comment by afritz1** [comment]
I noticed this, too, and @Carmina16 is right. `DZTTAV.IMG` is only 1088 bytes (32x34) instead of 4096 bytes (64x64) like most voxel textures.

I added a special case in commit 366ac3924ea34cc06d6ff6acffa50de5ebcc1376 by taking the 32x34 data and writing it into a 64x64 image, with an X offset of 32 so the texture matches the look of `DZTTEP.IMG`. I think those two signs were intended to look similar, and that's how they are now.

I also noticed them floating in the air when viewed from one direction, and that's a really easy fix to make in the renderer. I just wasn't sure at the time whether they were supposed to look identical on both sides or if they're supposed to be mirrored (and it seems the latter is the proper way).
